clients/client-java/src/main/java/com/datastrato/gravitino/client/BaseSchemaCatalog.java

```diff
@@ -194,7 +191,6 @@ public boolean dropSchema(NameIdentifier ident, boolean cascade) throws NonEmpty
     } catch (NonEmptySchemaException e) {
       throw e;
     } catch (Exception e) {
-      LOG.warn("Failed to drop schema {}", ident, e);
```

Comment:
After remove the log, we can't know why it failed.

Reply:
This is the issue in branch-0.5 that the semantic is not aligned.

Reply:
I think we can leave it as it is, We had better not change its original behavior in 0.5.1

Reply:
BTW, this will be fixed in 0.6.0